pyanaconda/ui/tui/spokes/software.py

```diff
@@ -90,6 +91,9 @@ def _initialize(self):
         # Apply the initial selection
         self._apply()
 
+        # report that the software spoke has been initialized
+        self.initialize_done()
```

Comment:
Good point - I'll add a `threadMgr.wait()` on the thread started by _apply() to make sure the software selection spoke is really initialized before initialize_done() is called.

The `_initialize()` method is already running in a thread so it should be OK. :)